packages/uiweb/src/lib/components/chat/MessageInput/MessageInput.tsx

```diff
@@ -509,8 +505,8 @@ console.log(checkIfMember(chatFeed, account!))
           )}
         </>
       ) : null}
-      {!!pgpPrivateKey &&
-        (((isRules ? verified : true) && isMember) ||
+      {
+        ((pushUser && (isRules ? verified : true) && isMember) ||
```

Comment:
instead of pushUser use Object.keys(PushUser || {}).length
Because its an object.